schemas/v3.1/schema.json

```diff
@@ -678,6 +691,7 @@
       }
     },
     "media-type": {
+      $comment": "https://spec.openapis.org/oas/v3.1.0#media-type-object",
```

Comment:
```suggestion
      "$comment": "https://spec.openapis.org/oas/v3.1.0#media-type-object",
```

Reply:
doh! I thought I'd proofread it carefully enough.

fixed!